### ISSUE TITLE
Inline PGP messages do not get detected by Oversec when showing in the GMail app.

### ISSUE THREAD

**Issue by oversecio** [oversecio/oversec_crypto#6]

**Comment by oversecio** [comment]
turns out that messages (in GMail as well as in K9) are shown in a WebView component.
Need to figure out how to get the text from that.
 (should be possible as TalkBack can read it, but simply adding "flagRequestEnhancedWebAccessibility" / "canRequestEnhancedWebAccessibility" in Oversec didn't do the trick :-( )

**Comment by oversecio** [comment]
works now
